src/longopt/psconvert_inc.h

```diff
@@ -25,74 +25,45 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'D', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'E', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'F', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'G', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'H', "",
-	          "",                  "",
+	{ 0, 'A', "adjust|autocrop|crop",
 	          "",                  "",
+	          "r,u",               "round,no_timestamp",
 		  GMT_TP_STANDARD },
-	{ 0, 'I', "",
-	          "",                  "",
+	{ 0, 'C', "gs_option|ghost_option", "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'D', "outdir|out_dir",    "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'E', "dpi",               "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'F', "outfile|out_name|prefix", "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'G', "gs_path|ghost_path", "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'H', "scale",             "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'I', "resize",
 	          "",                  "",
+	          "m,s,S",             "margins,size,scale",
 		  GMT_TP_STANDARD },
-	{ 0, 'L', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'M', "",
-	          "",                  "",
+	{ 0, 'L', "listfile|list_file", "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'M', "pslayer|ps_layer",
+	          "b,f",               "bg|background,fg|foreground",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'N', "",
-	          "",                  "",
+	{ 0, 'N', "bb_style|show_bbox",
 	          "",                  "",
+	          "f,g,k,p",           "fade,bg|background,fadecolor,pen",
 		  GMT_TP_STANDARD },
-	{ 0, 'P', "",
-	          "",                  "",
+	{ 0, 'Q', "anti_aliasing",
+	          "g,p,t",             "graphics,geopdf,text",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'Q', "",
-	          "",                  "",
-	          "",                  "",
+	{ 0, 'S', "gs_command|ghost_command", "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'T', "format|fmt",
+	          "b,e,E,f,F,j,g,G,m,t",
+		                       "bmp,eps,pageszeps,pdf,multipdf,jpeg,png,transpng,ppm,tiff",
+	          "m,q",               "mono|monochrome,quality",
 		  GMT_TP_STANDARD },
-	{ 0, 'S', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'T', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'W', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'Z', "",
-	          "",                  "",
+	{ 0, 'W', "esri|world_file",
```

Comment:
I don't see the need to add `esri`, which is the name of the company. `world_file` is like GDAL refers to it for example.